### DISCUSSION TITLE
clock with seconds in Win11 taskbar

### DISCUSSION THREAD

**Discussion by MrMega2004** [valinet/ExplorerPatcher#1164] [Ideas]
Hey there folks,
just wanted to mention that a clock with seconds would be real nice. Ever since I upgraded from Win10 to Win11 I missed this feature. Perhaps this could be a thing to add in a future update...

**Comment by Anixx** [comment]
What is this?

![изображение](https://user-images.githubusercontent.com/1271736/226710046-3f844dbc-d13c-4984-913c-71a31f149d40.png)

**Reply by MrMega2004** [reply]
yeah I noticed that but when you select Windows 11 as taskbar style the option is gone

**Comment by arcyc** [comment]
If you switch to Windows 10 Taskbar style and check this option and switch back to Windows 11 taskbar style it still works - seems to be a bug that it's not shown in System Tray settings when Windows 11 Taskbar style is selected.
